modules/ROOT/pages/index.adoc

```diff
@@ -17,15 +17,118 @@ Members of your organization can use Partner Manager to perform a variety of fun
 * Managing the activity of the business transactions with your trading partners
 * Optimizing backend services and performance
 
-== Prerequisites for Using Partner Manager
+== Before You Begin Using Partner Manager
 
-To use Partner Manager, you must be familiar with:
+Before you begin using Partner Manager, you must have:
 
-* EDI message formats
-* How to create DataWeave maps to perform data transformations
+* An Anypoint Platform user account with the Anypoint Partner Manager entitlement
+* The Partner Manager Administrator role, which assigns permission levels for to all users who need access to B2B message flows in their environments (production or  sandbox)
+* The required Mule vCore capacity to manage your B2B transaction workload needs
+* Storage for the B2B transmission payloads you send or receive, such as a database, Amazon S3, Azure Blob Storage, and so on
+* Licenses for the AS2, EDIFACT, or X12 connectors used in your B2B message flows
+* Anypoint Studio 7.4 or later
 
-To get started, ask your Anypoint Platform administrator to follow the setup instructions.
+Also, you must be familiar with EDI message formats and should know how to create DataWeave maps to use for performing data transformations.
+
+== Partner Manager Concepts
+
+To use Partner Manager, you must understand the following concepts:
+
+===== host
+
+Your Anypoint platform or business group. A Partner Manager implementation has a single host for which you define a host profile.
```

Comment:
I don't understand this first sentence. Should "platform" be init cap - is it referring to Anypoint Platform?